vendor/engines/secure_rooms/app/views/secure_rooms/facility_occupancies/index.html.haml

```diff
@@ -3,3 +3,37 @@
 
 = content_for :tabnav do
   = render "secure_rooms/shared/tabnav_occupancy"
+
+- if @order_details.present?
+  = form_tag(batch_update_facility_occupancies_path, :method => :post) do
+
+    %table.table.table-striped.table-hover.occupancies.old-table
+      = render partial: 'table_controls'
+      %thead
+        %tr
+          %th
+          %th= Order.model_name.human
+          %th= OrderDetail.model_name.human
+          %th= sortable 'entry_at', SecureRooms::Occupancy.human_attribute_name(:entry_at)
+          %th= sortable 'user_name', User.model_name.human
+          %th.order-note= sortable 'product_name', SecureRoom.model_name.human
+          %th= sortable 'payment_source', Account.model_name.human
+      %tbody
+        - @order_details.each do |od|
+          - order = od.order
+          - occ = od.occupancy
+          %tr
+            %td.centered= check_box_tag("order_detail_ids[]", od.id, false, {:class => 'toggle', :id => nil })
```

Comment:
We have haml-lint set up which should catch this, codeclimate doesn't have an engine for it, so you need to remember to run it manually.